apps/backend/pyproject.toml

```diff
@@ -10,25 +10,25 @@ readme = "README.md"
 requires-python = ">=3.12"
 license = { text = "MIT" }
 dependencies = [
-    "fastapi==0.95.1",
-    "uvicorn==0.22.0",
-    "sqlalchemy==2.0.12",
-    "pydantic==1.10.7",
-    "psycopg2-binary==2.9.6",
-    "alembic==1.10.4",
-    "python-dotenv==1.0.0",
+    "fastapi>=0.95.1",
+    "uvicorn>=0.22.0",
+    "sqlalchemy>=2.0.12",
+    "pydantic>=1.10.7",
+    "pydantic-extra-types>=2.0.0",
+    "psycopg2-binary>=2.9.6",
+    "alembic>=1.10.4",
+    "python-dotenv>=1.0.0",
```

Comment:
I had to add two dependencies to get this to run on my docker branch:

"phonenumbers>=8.13.1",
"pydantic[email]>=1.10.16",

Reply:
also, I needed to bump pydantic to 1.10.16 because 1.10.7 was throwing an error that they fixed